extensions/amp-story/1.0/pagination-buttons.css

```diff
@@ -59,9 +59,14 @@ amp-story:not([desktop]) .i-amphtml-story-button-container {
   pointer-events: none !important;
 }
 
-/* Navigation buttons. These are only displayed on desktop. */
-.i-amphtml-story-button-move {
-  display: none !important;
+/* Navigation button element. Visibly hidden on mobile but focusable by screen readers. */
+amp-story:not([desktop]) .i-amphtml-story-button-move {
+  width: 100% !important;
+  height: 100% !important;
+  border: none !important;
+  background-image: none !important;
```

Comment:
Doesn't `background: none` override the background image as well? Eg: https://codepen.io/mszylkowski/pen/NWrERLL

Reply:
Thanks for catching that.